src/main/java/com/mercadopago/resources/preference/PreferenceShipments.java

```diff
@@ -0,0 +1,36 @@
+package com.mercadopago.resources.preference;
+
+import java.math.BigDecimal;
+import java.util.List;
+import lombok.Getter;
+
+/** Shipments information from preference. */
+@Getter
+public class PreferenceShipments {
```

Comment:
```suggestion
public class PreferenceShipment {
```
Não deveria ser no singular?

Reply:
A API chama o [objeto de shipments](https://www.mercadopago.com.br/developers/pt/reference/preferences/_checkout_preferences/post#:~:text=marketplace_fee-,shipments,-Object) no plural, e a SDK de dotnet também trata dessa maneira, chamando o recurso no plural, então adotei o mesmo. Acha que é melhor alterarmos para o singular ou podemos deixar no plural mesmo?